scipy-1.0/paper.tex

```diff
@@ -122,6 +122,21 @@ \section*{Project organisation and community}
 
 \textbf{Governance}
 
+SciPy adopted an official Governance Document on August 3, 2017 (see PR\#6953). In summary,
```

Comment:
Why cite the PR rather than the document itself?

I think we're using BibLaTeX, in which case we can add this citation to the `.bib`:

@online{SciPyProjectGovernance,
  author = {The SciPy Community},
  title = {{SciPy} Project Governance},
  year = 2017,
  url = {https://docs.scipy.org/doc/scipy/reference/dev/governance/governance.html},
  urldate = {2018-06-01}
}
and then of course we can \cite{SciPyProjectGovernance} in the text.

If not, we can 
\usepackage{hyperref}
and it could be an @misc:
@misc{SciPyProjectGovernance,
  author = {The SciPy Community},
  title = {{SciPy} Project Governance},
  howpublished = {\url{https://docs.scipy.org/doc/scipy/reference/dev/governance/governance.html}},
  note = {Accessed: 2018-06-01}
}